Documentation/Concepts/Backend/SystemModules/Index.rst

```diff
@@ -1,77 +1,113 @@
+:navigation-title: System modules
 .. include:: /Includes.rst.txt
-.. index:: Modules; System
 .. _system-modules:
 
 ==============
 System modules
 ==============
 
-.. important::
+The backend modules in group system are only available to backend users with
+admin permissions.
 
-   This chapters cover modules that will only be available for backend users
-   with admin access privileges.
+Some modules are only available when you have an optional system extension
```

Comment:
What is meant by "optional system extension" here?

Reply:
One that you can install but don't have to install. Maybe put it in the glossary: 

Optional system extension: A system extension in TYPO3 that is not installed by default but can be installed as needed to provide additional functionality.